src/main/java/io/wispforest/owo/ui/core/Sizing.java

```diff
@@ -90,6 +91,229 @@ public static Sizing expand(int percent) {
+            return Sizing.fixed(SIZING_RANDOM.nextInt(0, max));
+        }
+
+        /**
+         * Generate a random fixed sizing instance with a value between 0 and 100
+         *
+         * @return A random sizing instance
+         * @author chyzman
+         */
+        public static Sizing fixed() {
+            return Sizing.fixed(SIZING_RANDOM.nextInt(0, 100));
+        }
+
+        /**
+         * Generate a random content sizing instance with a padding value between {@code min} and {@code max}
+         *
+         * @param min The minimum value
+         * @param max The maximum value
+         * @return A random sizing instance
+         * @author chyzman
+         */
+        public static Sizing content(int min, int max) {
+            return Sizing.content(SIZING_RANDOM.nextInt(min, max));
+        }
+
+        /**
+         * Generate a random content sizing instance with a padding value between 0 and {@code max}
+         *
+         * @param max The maximum value
+         * @return A random sizing instance
+         * @author chyzman
+         */
+        public static Sizing content(int max) {
+            return Sizing.content(SIZING_RANDOM.nextInt(0, max));
+        }
+
+        /**
+         * Generate a random content sizing instance with a padding value between 0 and 100
+         *
+         * @return A random sizing instance
+         * @author chyzman
+         */
+        public static Sizing content() {
+            return Sizing.content(SIZING_RANDOM.nextInt(0, 100));
+        }
+
+        /**
+         * Generate a random sizing instance with a value between {@code min} and {@code max}
+         *
+         * @param min The minimum value
+         * @param max The maximum value
+         * @return A random sizing instance
+         * @apiNote Can crash if put on a component that isn't a container
+         * @author chyzman
+         */
+        public static Sizing random(int min, int max) {
+            return switch (SIZING_RANDOM.nextInt(0, 4)) {
+                case 0 -> fill(min, max);
+                case 1 -> expand(min, max);
+                case 2 -> fixed(min, max);
+                case 3 -> content(min, max);
+                default -> throw new IllegalStateException("Unexpected value: " + SIZING_RANDOM.nextInt(0, 4));
+            };
+        }
+
+        /**
+         * Generate a random sizing instance with a value between 0 and {@code max}
+         *
+         * @param max The maximum value
+         * @return A random sizing instance
+         * @apiNote Can crash if put on a component that isn't a container
+         * @author chyzman
+         */
+        public static Sizing random(int max) {
+            return random(0, max);
+        }
+
+        /**
+         * Generate a random sizing instance with a value between 0 and 100
+         *
+         * @return A random sizing instance
+         * @apiNote Can crash if put on a component that isn't a container
```

Comment:
More like, any component that doesn't support content sizing - scroll containers come to mind (at least on one of their axes)